maker/src/routes.rs

```diff
@@ -276,6 +276,30 @@ pub async fn get_metrics<'r>(_auth: Authenticated) -> Result<String, HttpApiProb
     Ok(metrics)
 }
 
+#[derive(Debug, Clone, Copy, Deserialize)]
+pub struct RolloverConfig {
+    is_accepting_rollovers: bool,
+}
+
+#[rocket::post("/rollover-config", data = "<config>")]
+#[instrument(name = "POST /rollover-config", skip(maker), err)]
+pub async fn update_rollover_configuration(
```

Comment:
What is the motivation of this change? Because we remove the `accept` step we need another way to just stop accepting rollovers? What's the use case for this?

Reply:
@bonomat maybe you can explain why this feature is needed. I feel we never really turned rollovers off, so I wonder if this feature is actually justified.

Reply:
@bonomat I'm merging this as is. But we can remove this feature later if it's deemed unnecessary.

Reply:
Sorry, I missed this comment. We might indeed not need this feature.